portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -7136,6 +7207,8 @@ private void _sendNotificationEmail(
 	private static final Log _log = LogFactoryUtil.getLog(
 		UserLocalServiceImpl.class);
 
+	private static final AuditRouter _auditRouter =
+		AuditRouterUtil.getAuditRouter();
```

Comment:
Better to use AuditRouterUtil#route directly in the relevant code. Because otherwise this may end up being null if `UserLocalServiceImpl` initializes earlier than the `AuditRouter` OSGi service.